demos/test-app/src/index.tsx

```diff
@@ -134,11 +134,38 @@ const updateDelegationView = ({
   }
 
   if (identity instanceof DelegationIdentity) {
-    delegationEl.innerText = JSON.stringify(
-      identity.getDelegation().toJSON(),
-      undefined,
-      2,
-    );
+    const delegation = identity.getDelegation();
+    let jsonResult;
+    try {
+      // `toJSON` is failing due to `bytesToHex(delegation.publicKey)`
+      // `bytesToHex` expects a Uint8Array and delegation.publicKey is a ArrayBuffer
+      jsonResult = {
```

Comment:
Should this new code be unit tested?

Reply:
This is a test application, and it's also used in E2E tests. I'm not sure we need more tests for this part.